### DISCUSSION TITLE
Is Supabase suitable for my project?

### DISCUSSION THREAD

**Discussion by Posoroko** [supabase/supabase#1064] [Questions]
Hello everyone!

I'm still new to web developpement, I started learning 6 months ago. I've learnt a lot about html, css and JS and I now want to build a bigger project.  I've spent most of my time learning by my self, but I really invested my self in this new passion!

I recently realised that these three languages are of very little use when it comes to bigger project. I started learning Node.js, express, vue.js,  and found out that the web pages I was building are net the pages people are building in "the real world".
so, I'm now looking around trying to figure out what the heck I'm suppose to learn and use to build my project!!!

My project is quite simple: A web app where people can create an account, put in their info, post photos and write blog articles. They can also check other people's photos and make searches to finds specific photos using keywords. It conserns a very specific community of people, but "outsiders" could also come as visitors to check out the latest news, articles and photos. For those wandering, it has nothing to do with porn... :-)

When learning nodejs, I found out about 'serve side rendering' and it seemed awesome to me. Being able to create custom html pages based on the user infos, account or searches seems pretty important when it comes to non static web apps.

Qestions:

Is custom resdering possible with Supabase? The clound functions are not available yet... And they seem qui te necessary to me. Are there other ways to achieve the same result without them?

How can we work with the database in a safe way without server function?

question on another topic but very importnat for me:

What tools should I learn? Everytime I watch a course or a tutorial about one tool, I hear about 10 new ones that I've never heard before... It seems like it will never stop! I feel like I need to focus on a set of tools and start building!!! but what tools!!!

My message is a bit of a mess, and I 'm sorry for that! I really confused about al this and I've been going around in cercles for 2 months now trying to figure things out, and I just can't find answers to help me out.

Thank you sur yout precious time, 

Eric

**Comment by davidtew** [comment]
Here is another tool for you to check out.  https://draftbit.com/   whose blurb is : "Build your app, visually. Create, customize, and launch mobile apps all from your browser. Source code included."  

I am suggesting this to you since:
 - they have recently demoed connecting their tool to supabase
 - their team are online to give you support - so I'm betting you can probably can get some (free) answers to some of your questions very quickly there. 
 - perhaps it is a good tool for a beginner to use to actually create a powerful product, much like you are hoping to do

 If you do investigate draftbit.com, please write your experience back here, and let us know how it went.
